examples/cookbook/src/lib.rs

```diff
@@ -181,9 +180,9 @@ mod tests {
 
             // we don't transfer the full base asset so we can cover fees
             let output = if id == *consensus_parameters.base_asset_id() {
-                wallet_1.get_asset_outputs_for_amount(wallet_2.address(), id, amount / 2)
+                wallet_1.get_asset_outputs_for_amount(wallet_2.address(), id, (amount / 2) as u64)
```

Comment:
curious, does get_balances return an amount which is not u64?

Reply:
yes the type is `u128` as we can have multiple coins (`u64`) so that we do not overflow.